src/components/home/WhyJoin.jsx

```diff
@@ -1,5 +1,35 @@
+import Title from "../Title";
+import { FOCUS } from "@/data/focus";
+import Focus from "./Focus";
+import Picture from "../Picture";
 const WhyJoin = () => {
-  return <div>WhyJoin</div>;
+  return (
+    <div className="p-16 flex flex-col items-center">
+      <div className="absolute w-1/3 aspect-square bg-ewb-blue-100/40 rounded-full -translate-y-14" />
+      <Title subTitle="GET INVOLVED AND MAKE AN IMPACT!" title="WHY JOIN EWB" />
```

Comment:
go into the title component and add a z-10 to the className, because sometime the title is behind the circle